awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+
+import { mountWithContexts } from '../../../../testUtils/enzymeHelpers';
+
+import OrganizationExecEnvListItem from './OrganizationExecEnvListItem';
+
+describe('<OrganizationExecEnvListItem/>', () => {
+  let wrapper;
+  const executionEnvironment = {
+    id: 1,
+    image: 'https://registry.com/r/image/manifest',
```

Comment:
do we need to add `name` and `pull` as well?

Reply:
Good idea - done